esp32HTTPrequest.cpp

```diff
@@ -280,16 +282,27 @@ size_t  esp32HTTPrequest::_send(const char* body, size_t len){
     }
     delete _headers;
     _headers = nullptr;
+    _requestLen = len;
     if(len){
-        esp_http_client_set_post_field(_client, body, len);
+      esp_http_client_set_post_field(_client, body, len);
+    }
+    bool isTLS = strcmp(_URL->scheme, "HTTPS") == 0;
+    if(isTLS){
+        xSemaphoreTake(TLSlock_S, portMAX_DELAY);
```

Comment:
Thanks for update, 
Not sure what these xSemaphore lines does, but I had to remove it for this version to work.